### PR TITLE
Workload dashboards

### PR DESCRIPTION
**Pull request by csmarchbanks**

### CODE REVIEW

**Comment by cboggs** [comment]
At a glance, :+1:. Looks like jsonnet. :-D

Rules make sense, dashboard is presumably... a... dashboard...? Will look at screenshots when available. :-D

File: rules/rules.libsonnet [hunk 1]
```diff
@@ -373,19 +419,19 @@
             ||| % $._config,
           },
           {
-              record: 'node:node_inodes_free:',
-              expr: |||
+            record: 'node:node_inodes_free:',
```
Comment (ellisda) [inline rules/rules.libsonnet:422]: These just look like whitespace changes, right?

File: rules/rules.libsonnet [hunk 2]
```diff
@@ -67,6 +67,52 @@
               )
             ||| % $._config,
           },
+          // workload aggregation for deployments
+          {
+            record: 'mixin_pod_workload',
```
Comment (ellisda) [inline rules/rules.libsonnet:72]: This is slick, no regex parsing needed

File: dashboards/resources.libsonnet [hunk 3]
```diff
@@ -172,6 +172,211 @@ local g = import 'grafana-builder/grafana.libsonnet';
...
+        workload: {
+          alias: 'Workload',
+          link: '%(prefix)s/d/%(uid)s/k8s-resources-workload?var-datasource=$datasource&var-cluster=$cluster&var-namespace=$namespace&var-workload=$__cell' % { prefix: $._config.grafanaPrefix, uid: std.md5('k8s-resources-workload.json') },
+        },
+        workload_type: {
+          alias: 'Workload Type',
+        },
+      };
+
+      local cpuUsageQuery = |||
```
Comment (ellisda) [inline dashboards/resources.libsonnet:186]: This is clean style, with the queries defined in locals, I like it. It's a little different style than other grafana-builder dashboards, but looks clean.

File: config.libsonnet [hunk 4]
```diff
@@ -35,6 +35,8 @@
       'nodes.json': 'kcb9C2QDe4IYcjiTOmYyfhsImuzxRcvwWC3YLJPS',
       'pods.json': 'AMK9hS0rSbSz7cKjPHcOtk6CGHFjhSHwhbQ3sedK',
       'statefulset.json': 'dPiBt0FRG5BNYo0XJ4L0Meoc7DWs9eL40c1CRc1g',
+      'k8s-resources-workloads-namespace.json': 'L29WgMrccBDauPs3Xsti3fwaKjMB6fReufWj6Gl1',
```
Comment (ellisda) [inline config.libsonnet:38]: yea! generated UIDs that don't go anywhere :smile:  (LGTM)

**Review by ellisda — APPROVED** [review]

File: rules/rules.libsonnet (hunk 1 above)
Comment (csmarchbanks) [reply rules/rules.libsonnet:422]: According to `?w=1`, yes these are all just whitespace changes.

**Pull request merged**